docs/platform/release-notes/tizen-6-0-m1.md

```diff
@@ -0,0 +1,692 @@
+
+- Generic platform status dump tool has been provided to dump the status of Tizen platform as unified interface.
+- System status dump tool has been enhanced to support vendor-specific command line flag.
+- C# call stack analyzer and interactive debugging mode have been added to Tizen Callstack Analyzer.
+
+### Fixes
+
+- Crash garbage due to sudden power failure has been removed.
+
+
+## System (Base)
+
+### New and changed features
+
+- Open source
+  - aspell 
+    - Version 0.60.5.1 has been upgraded to version 0.60.8.
+  - boost 
+    - Version 1.65.1 has been upgraded to version 1.71.0.
+  - bzip 
+    - Version 1.06 has been upgraded to version 1.08.
+  - dos2unix
+    - Version 7.4.0 has been upgraded to version 7.4.1.
+  - expat
+    - Version 2.2.7 has been upgraded to version 2.2.9.
+  - glib 
+    - Version 2.52.2 has been upgraded to version 2.62.3.
+  - hostname 
+    - Version 3.2 has been upgraded to version 3.23.
+  - icu 
+    - Version 63.2 has been upgraded to version 65.
+  - iniparser 
+    - Version 3.2 has been upgraded to version 4.1.
+  - jsoncpp
+    - Version 1.84 has been upgraded to version 1.92.
+  - Json-glib 
+    - Version 1.4.2 has been upgraded to version 1.4.4.
+  - kbd
+    - Version 2.0.4 has been upgraded to version 2.2.0.
+  - leveldb
+    - Version 1.2 has been upgraded to version 1.22.
+  - libarchive
+    - Version 3.3.3 has been upgraded to version 3.4.0.
+  - libevent
+    - Version 2.1.8 has been upgraded to version 2.1.11.
+  - libsolv
+    - Version 0.6.35 has been upgraded to version 0.7.6.
+  - libxml2
+    - Version 2.9.7 has been upgraded to version 2.9.10.
+  - libxslt
+    - Version 1.1.32 has been upgraded to version 1.1.34.
+  - libzypp
+    - Version 14.27.0 has been upgraded to version 17.14.0.
+  - meson
+    - Version 0.51.2 has been upgraded to version 0.52.1.
+  - ncurse
+    - Version 6.1 has been upgraded to version 6.1-20191207.
+  - ninja
+    - Version 1.8.2 has been upgraded to version 1.9.0.
+  - python
+    - Version 2.7.15 has been upgraded to version 2.7.17.
+  - sqlite
+    - Version 3.29.0 has been upgraded to version 3.31.1.
+  - tzdata
+    - Version 2019a has been upgraded to version 2019c.
+  - util-linux
+    - Version 2.3 has been upgraded to version 2.34.
+  - zypper
+    - Version 1.11.11 has been upgraded to version 1.14.30.
+  - perl-XML-LibXML (2.0134) has been added.
+  - perl-XML-NamespaceSupport (1.12) has been added.
+  - perl-XML-SAX (1.02) has been added.
+- The capi-base-utils API set has been expanded.
+  - Add
+    - unorm2 : Unicode normalization module.
+  - New
+    - unumsys : Numbering systems module.
+    - utext : Text abstraction module.
+    - uscript : Unicode Script module.
+    - uidna : Internationalizing Domain Names in Applications (IDNA) module.
+    - ucnv : Character conversion module.
+    - ucnvsel : Converter selector module.
+    - ucsdet : Detecting the charset or encoding of character data module.

```

Comment:
Request you to please remove the space before the colon (:):

- unorm2: Unicode normalization module.
  - New
    - unumsys : Numbering systems module.
    - utext: Text abstraction module.
    - uscript: Unicode Script module.
    - uidna: Internationalizing Domain Names in Applications (IDNA) module.
    - ucnv: Character conversion module.
    - ucnvsel: Converter selector module.
    - ucsdet: Detecting the charset or encoding of character data module.

Reply:
will accept